src/main/java/dev/jbang/cli/Template.java

```diff
@@ -263,6 +313,22 @@ private static void printTemplate(PrintStream out, String catalogName, Catalog c
 				}
 			}
 		}
+		if (showProperties && template.properties != null) {
+			for (Map.Entry<String, TemplateProperty> entry : template.properties.entrySet()) {
+				out.print(Util.repeat(" ", indent));
+				StringBuilder propertyLineBuilder = new StringBuilder()
+																		.append(Util.repeat(" ", indent + 4))
+																		.append(ConsoleOutput.cyan(entry.getKey()))
+																		.append(" = ");
+				if (StringUtils.isNotBlank(entry.getValue().getDescription())) {
```

Comment:
Given the changes I suggested for the `TemplatePropertyConverter` I'd simply check for `null` here.